tools/ratbag-command.c

```diff
@@ -145,12 +147,27 @@ usage(void)
 	       "\n"
 	       "  switch-etekcity		Switch the Etekcity mouse active profile\n"
 	       "\n"
+	       "LED Commands:\n"
+	       "  LED commands work on the given profile, or on the\n"
+	       "  active profile if none is given.\n"
+	       "\n"
+	       "  led N {COMMAND}\n"
+	       "                      get      print the LED properties\n"
+	       "                      set {COMMAND}      sets LED properties\n"
```

Comment:
this may be the github interface mis-displaying them but can we align the print/sets on the same column please?